docs/developing/dev_environment/local_proxy_remote_solana.md

```diff
@@ -64,48 +64,56 @@ The Neon EVM address is registered inside `neonlabsorg/proxy`, so the Proxy know
 #### Create and run services with Docker Compose
 In order to create and run these services: 
 
-1. Create a "keys" folder and put your allowlisted key into it. Note that the file with the Operator key should be named `id.json`.
-```bash
-mkdir keys
-mv {PATH_TO_WHITELISTED_KEYS} keys/
-```
+1. Create a file `id.json` for storing an Operator key.
+   Put Solana private key to the `id.json`.
 
 2. Set the following environment variables
    - `EVM_LOADER`, i.e. the contract address for Neon EVM
      - For Devnet/Testnet, use: `eeLSJgWzzxrqKv1UxtRVVH8FX3qCQWUs9QuAjJpETGU`
      - For Mainnet, use: `NeonVMyRX5GbCrsAHnUwx1nYYoJAtskU1bWUo6JGNyG`
    - `SOLANA_URL`
      - Refer to the [RPC Endpoints table](#rpc-endpoints)
-   - `REVISION` - neon proxy revision
+   - `VERSION` - neon proxy revision
+   - `SOLANA_KEY_FOR_EVM_CONFIG` - operator key, should be a valid solana public key with SOLs on it
 
-For example,
+Example for devnet configuration:
 ```bash
-export EVM_LOADER=eeLSJgWzzxrqKv1UxtRVVH8FX3qCQWUs9QuAjJpETGU
-export SOLANA_URL=http://api.devnet.solana.com/
-export REVISION=latest
+EVM_LOADER=eeLSJgWzzxrqKv1UxtRVVH8FX3qCQWUs9QuAjJpETGU
+VERSION=v1.13.20
+SOLANA_URL=http://api.devnet.solana.com/
+SOLANA_KEY_FOR_EVM_CONFIG=<your solana public key>
 ```
 
-3. Download the `docker-compose` [file](https://github.com/neonlabsorg/proxy-model.py/blob/develop/docker-compose/docker-compose-remote-solana.yml). This file should be placed in the same folder with the `keys/` directory.
+3. Download the `docker-compose` [file](https://github.com/neonlabsorg/neon-proxy.py/blob/b3bd1a298f3a437cb48379f348ed71268af382cc/docker-compose.yml). This file should be placed in the same folder with the `keys/` directory.
```

Comment:
It it not enough just to download docker-compose file. It will not start without db scheme sql file and you have to create a directory for indexer data. I put all these instructions into a draft doc in notion, please add it here